native/native_image.go

```diff
@@ -69,9 +70,14 @@ func (n NativeImage) Contribute(layer libcnb.Layer) (libcnb.Layer, error) {
 	if err != nil {
 		return libcnb.Layer{}, fmt.Errorf("unable to process arguments\n%w", err)
 	}
+
+	if !slices.Contains(arguments, "--auto-fallback") && !slices.Contains(arguments, "--force-fallback") {
```

Comment:
Is there a test case that includes `--auto-fallback` or `--force-fallback`?  I don't see it in the commit log.